README.md

```diff
@@ -1,2 +1,94 @@
 # npmx-zephyr
-npmx integration with Zephyr
+
+<p align="center">
+    <img src="doc/_static/images/npm_family_logo_background.png" alt="NPM LOGO" width="200" height="200">
+</p>
+
+## Overview
+npmx-zephyr repository provides an example of [npmx library][npmx_repo] integration with the [Zephyr][zephyr_site] RTOS environment. This can be used as a starting point for an integration with other RTOS or bare-metal environments.
+
+For the user's convenience, this repository includes also a variety of sample applications demonstrating how to use the [npmx drivers][npmx_drivers].
+
+## Supported devices:
+ -    nPM1300 Engineering B
+
+## How to use
+
+### Repository initialization
+The recommended way to clone the repositories is by using the `west` tool. Information about the installation, available commands, configuration and more can be found in the [Zephyr's west][west_doc] tool documentation.
+```bash
+west init -m https://github.com/NordicSemiconductor/npmx-zephyr
+cd npmx-zephyr
+west update
```

Comment:
`mkdir npmx`
`cd npmx`
`west init -m https://github.com/NordicSemiconductor/npmx-zephyr`
`cd npmx-zephyr`
`west update`

`west init` command initializes some repos in folders above the folder where it is invoked. Maybe it's worth creating a parent folder first?

Reply:
You are right, it's more convenient way